src/jdk.compiler/share/classes/com/sun/tools/javac/launcher/Main.java

```diff
@@ -407,37 +410,70 @@ private String compile(Path file, List<String> javacOpts, Context context) throw
         return mainClassName;
     }
 
+    private static boolean isPublic(Method method) {
+        return method != null && Modifier.isPublic(method.getModifiers());
+    }
+
+    private static boolean isStatic(Method method) {
+        return method != null && Modifier.isStatic(method.getModifiers());
+    }
```

Comment:
It looks like these new methods aren't being used.

Reply:
Remnants - removing